sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -660,19 +663,19 @@ namespace Azure { namespace Core { namespace Http {
     // ===== Methods used to build HTTP response =====
 
     /**
-     * @brief Add HTTP header to the #Azure::Core::Http::RawResponse.
+     * @brief Set an HTTP header to the #Azure::Core::Http::Request.
      *
      * @remark The \p name must contain valid header name characters (RFC 7230).
      *
-     * @param name The name for the header to be added.
-     * @param value The value for the header to be added.
+     * @param name The name for the header to be set or added.
+     * @param value The value for the header to be set or added.
      *
      * @throw if \p name contains invalid characters.
      */
-    void AddHeader(std::string const& name, std::string const& value);
+    void SetHeader(std::string const& name, std::string const& value);
 
     /**
-     * @brief Add HTTP header to the #Azure::Core::Http::RawResponse.
+     * @brief Set an HTTP header to the #Azure::Core::Http::Request.
```

Comment:
Same here and below. This isn't the `Request`, but rathe the `RawResponse`.

Reply:
I should take care more attention to the doxygen comments. :(
Here's the fix: https://github.com/Azure/azure-sdk-for-cpp/pull/1783

Thank you for checking Ahson!  soo cool! 👍